enforce-policies/README.md

```diff
@@ -52,14 +52,16 @@ In this exercise we'll use the denier adapter.
 
 2. Verify that the service is denied:
 
-   In [this exercise](../create-ingress-gateway/README.md), we created the Ingress Gateway resource. Make sure the $INGRESS_IP environment variable is still present. Then in the terminal, try:
+   In [Exercise 5](../exercise-5/README.md), we created the Ingress resource. Make sure the $INGRESS_IP environment variable is still present. Then in the terminal, try:
```

Comment:
the repo targeted by this PR does not have a `../exercise-5/README.md` file nor does the PR create one so this needs to be amended to the original.

Reply:
LGTM

Reply:
@timroster README.md file was updated. I also replace the .yaml file in the README.md file